src/notarytool.ts

```diff
@@ -43,17 +49,25 @@ function authorizationArgs(rawOpts: NotaryToolCredentials): string[] {
 
 async function getNotarizationLogs(opts: NotaryToolStartOptions, id: string) {
   try {
-    const logResult = await spawn('xcrun', ['notarytool', 'log', id, ...authorizationArgs(opts)]);
+    const logResult = await runNotaryTool(
+      ['log', id, ...authorizationArgs(opts)],
+      opts.notarytoolPath,
+    );
     d('notarization log', logResult.output);
     return logResult.output;
   } catch (e) {
     d('failed to pull notarization logs', e);
   }
 }
 
-export async function isNotaryToolAvailable() {
-  const result = await spawn('xcrun', ['--find', 'notarytool']);
-  return result.code === 0;
+export async function isNotaryToolAvailable(notarytoolPath?: string) {
+  if (typeof notarytoolPath !== 'undefined') {
```

Comment:
```suggestion
  if (notarytoolPath !== undefined) {
```

No need for `typeof` here, `notarytoolPath` is declared as a parameter.